.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        platform: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - uses: actions/cache@v1
+      id: cache-dependencies
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Install Dependencies
+      if: steps.cache-dependencies.outputs.cache-hit != 'true'
+      run: go get ./...
```

Comment:
This is technically correct, but I believe is too naive. Unfortunately, we rewrite our imports in the go.mod file in samplr, which causes go get at the root of the repository to fail.

```
package github.com/GoogleCloudPlatform/devrel-services/git-go: cannot find package "github.com/GoogleCloudPlatform/devrel-services/git-go" in any of:
	/opt/hostedtoolcache/go/1.13.7/x64/src/github.com/GoogleCloudPlatform/devrel-services/git-go (from $GOROOT)
	/home/runner/go/src/github.com/GoogleCloudPlatform/devrel-services/git-go (from $GOPATH)
```

In `samplr/go.mod`:
```
replace github.com/GoogleCloudPlatform/devrel-services/git-go => ./git-go
```
With no root-level go.mod to perform these rewrites, `go get` is failing.

There shouldn't be any issues with moving it to the root of the repository so these imports work, but it wouldn't solve the other rewrites we do. It might be better to iterate through each directory and go get from there.

Reply:
Done